projects/core/src/lib/map/bookmark/bookmark.proto

```diff
@@ -1,9 +1,23 @@
-// The bookmark_pb.ts is generated from this file by the bin/prebuild.js script that is run
-// automatically when running 'npm run build'
+// Regenerate bookmark_pb.ts by using `npm run generate-protobuf`
 
 syntax = "proto3";
 package bookmark;
 
+// The bookmark fragments with binary (protobuf) data. Because fragments may use long strings to refer to the same app layer or node ids,
+// these are combined in this message and compressed once instead of compressed separately
+message BinaryBookmarkFragments {
+  // Versioning to allow for backwards compatibility for old bookmarks
+  uint32 version = 1;
+  repeated BookmarkFragment fragments = 3;
```

Comment:
Might be useful to use `map<string, bytes> fragments = 2;` here instead? Though if you want consistent bookmarks for consistent state, keeping the list but having them be sorted when put into it would be better..

Reply:
The fragments are added in sort-order of the 'identifier' in the for-loop:

```typescript
    for(const id of [...fragmentById.keys()].sort()) {
       ...
       binaryFragments.fragments.push(new BookmarkFragment({
    }
```
Didn't change this compared to the previous code. The bookmark should ideally be 'stable' in that the same bookmark should always be encoded the same.